oc_checksums_worker/checksums_worker.py

```
@@ -6,16 +6,57 @@
+                logging.debug("message type: [%s]", msg_type)
+                if msg_type == "register_file":
+                    fl = msg[1][0][0], msg[1][0][1], msg[1][2]
+                    logging.debug("File location: [%s]", fl)
+                    citype = msg[1][1]
+                    logging.debug("citype is [%s]", citype)
+                    logging.debug("executing register_file")
+                    self.register_file(fl, citype)
+                else:
+                    logging.error("unknown msg_type")
+            except Exception as e:
+                logging.exception("An exception occured: %s", e)
```

Ruff (0.15.10)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)